WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static async Task<(Height height, uint256 hash)> WaitForNewBlockAsync(thi
 		/// Waits for a specific new block and returns useful info about it.
 		/// </summary>
 		/// <param name="blockHash">Block hash to wait for</param>
-		/// <param name="timeout">(int, optional, default=0) Time in milliseconds to wait for a response. 0 indicates no timeout.</param>
+		/// <param name="timeout">Time in milliseconds to wait for a response. (int, optional, default=0). 0 indicates no timeout.</param>
```

Comment:
```suggestion
		/// <param name="timeout">Time in milliseconds to wait for a response. 0 indicates no timeout.</param>
```

What I removed here does not provide any information that the IDEA doesn't provide it anyway.